docs/contributing.rst

```diff
@@ -0,0 +1,181 @@
+Contributing
+============
+
+.. _developer-installation:
+
+Developer Installation
+----------------------
+
+If you want to help contribute to ``trio-websocket``, then you will need to
+install additional dependencies that are used for testing and documentation. The
+following sequence of commands will clone the repository, create a virtual
+environment, and install the developer dependencies.
+
+::
+
+    $ git clone git@github.com:HyperionGray/trio-websocket.git
+    $ cd trio-websocket
+    $ python3 -m venv venv
```

Comment:
First I've heard of venv-- it doesn't seem that popular?  pipenv seems to be the future and is well suited for library dev environments, so I've filed #61

Reply:
I saw that. I like the idea but I don't know pipenv very well. I'm trying it out on another project before I integrate it over here.